tests/data/test_subject.py

```diff
@@ -159,9 +159,12 @@ def test_copy_subject(self):
 
     def test_copy_subclass(self):
         class DummySubjectSubClass(tio.data.Subject):
-            pass
+            def __init__(self, attr_1, attr_2):
+                super().__init__(attr_1=attr_1, attr_2=attr_2)
```

Comment:
Maybe we don't need to specify parameters explicitly here.

```suggestion
            def __init__(self, **kwargs):
                super().__init__(**kwargs)
```